en/theme/material/partials/footer.html

```diff
@@ -64,7 +64,14 @@
             {{ config.copyright }}
           </div>
         {% endif %}
-        &copy;<script>document.write(new Date().getFullYear());</script> <a href="https://wso2.com/">WSO2 LLC.</a>
+        <div class="md-footer-copyright__content">
+          <div>
+              Copyright &copy; <a href="https://wso2.com/">WSO2</a> LLC 2015
```

Comment:
As this is the master branch is this date correct? Shouldn't it state 2023?

Reply:
hmm.. I left it as it is and added an item to the release checklist [1]. But +1 to change it to 2023 now it self. Will update that..

[1] https://docs.google.com/spreadsheets/d/1t7zXEWyYmbv8PxmG_ecPSd25BVHxumFw8k-_CZ92pFI/edit#gid=0